manifests/g/Genymobile/scrcpy/3.0.1/Genymobile.scrcpy.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Genymobile.scrcpy
+PackageVersion: 3.0.1
+InstallerType: zip
+NestedInstallerType: portable
+ArchiveBinariesDependOnPath: true

```

Just a stupid nitpick with the ordering of `ReleaseDate` that makes no real functional difference.

```suggestion
NestedInstallerType: portable
ReleaseDate: 2024-12-04
ArchiveBinariesDependOnPath: true
```